### ISSUE TITLE
kyivindependent.com: nuisance

### ISSUE THREAD

**Issue by BillyBobMcInbredYahoo** [uBlockOrigin/uAssets#25063]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://kyivindependent.com
```

### Description

Newsletter pop-ups & merchandise pop-ups aren't blocked on the Kyiv Independent website.

### Other extensions used

I still don't care about cookies

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml


```
</details>

**Comment by mapx-** [comment]
enable `EasyList – Annoyances`

**Comment by BillyBobMcInbredYahoo** [comment]
> enable `EasyList – Annoyances`

This does NOT fix the poblem.

**Comment by mapx-** [comment]
provide screenshots with the stuff still unblocked (and the steps to get it)

**Comment by BillyBobMcInbredYahoo** [comment]
> provide screenshots with the stuff still unblocked (and the steps to get it)

The attached screenshot shows a popup for Kyiv Independent merchanise for sale.

1) Go to the https://kyivindependent.com
2) Click on any article
3) Scroll down the page and after a while the popup will appear (sometimes it's the merch popup, other times it a popup so people can provide their email address to get email notifications of new articles)


![Kyiv Independent screenshot showing merch popup](https://github.com/user-attachments/assets/fdce67b3-c6f6-4069-a67e-a020e3290a38)

**Comment by mapx-** [comment]
I get only the first one, add:

`kyivindependent.com###popup-banner-subscription-store`

@ryanbr

**Comment by BillyBobMcInbredYahoo** [comment]
> kyivindependent.com###popup-banner-subscription-store

Of course, this can be done manually, but I'd like this to be added to a filter list so it's done automatically.

Also, when I used "Enter element picker mode", the following was added to "My filters":
! 29 Aug 2024 https://kyivindependent.com
kyivindependent.com##.popupBannerSubscribeStore__content

Why does it look different to your:
kyivindependent.com###popup-banner-subscription-store

**Comment by mapx-** [comment]
see above, it was already fixed (by ryanbr)

**Comment by BillyBobMcInbredYahoo** [comment]
> see above, it was already fixed (by ryanbr)

But it's NOT fixed.

I deleted
kyivindependent.com##.popupBannerSubscribeStore__content
from "My filters" (which DID work  btw), but now the merch popup still appears.

What is going on???

**Comment by stephenhawk8054** [comment]
@BillyBobMcInbredYahoo The filter will need some time to auto-update to users. Just wait. You can use that custom filter in the meantime.

**Comment by BillyBobMcInbredYahoo** [comment]
OK thank you, the issue has now been fixed.

I've deleted
kyivindependent.com##.popupBannerSubscribeStore__content
and the EasyList filter list is now blocking the merch popups on kyivindependent.com

**Comment by BillyBobMcInbredYahoo** [comment]
Thanks to all, especially ryanbr (can't see a way to contact him directly).

**Comment by BillyBobMcInbredYahoo** [comment]
MasterKia #25166 is a **different** popup issue that hasn't been resolved. Please don't close issues when they've not been resolved.

**Comment by stephenhawk8054** [comment]
@stephenhawk8054 Just continue in your old thread. All volunteers are notified with new comments.

**Comment by BillyBobMcInbredYahoo** [comment]
> @stephenhawk8054 Just continue in your old thread. All volunteers are notified with new comments.

The old thread (i.e. this thread) (#25063) relates to an issue/popup that has been resolved.

Why would I need to comment in an old thread/this thread about a **new** issue? That makes no sense.

As I said, this thread #25063 has been resolved.

It's #25166 that hasn't yet been resolved, but for some inexplicable reason it's been labelled as a duplicate and closed as 'not planned' (whatever that means). How do I get it unlabelled as a duplicate and reopened?

To be clear, both threads relate to popups on kyivindependent.com, but they're two **different** popups. One popup has been resolved/blocked, the other hasn't.

**Comment by stephenhawk8054** [comment]
Just comment any nuisances in the same nuisance thread.

**Comment by BillyBobMcInbredYahoo** [comment]
> Just comment any nuisances in one nuisance thread.

OK.

Go to https://kyivindependent.com/

At the top of the screen there's a popup entitled "The Kyiv Independent launches travel show".

Could someone please add a filter entry to block this popup?

**Comment by BillyBobMcInbredYahoo** [comment]
Why was this comment marked as 'spam'? It's not spam, it's a **clarification**.

**Comment by BillyBobMcInbredYahoo** [comment]
Why is no-one addressing the new issue I raised above?

As I said:

Go to https://kyivindependent.com/

At the top of the screen there's a popup entitled "The Kyiv Independent launches travel show".

Could someone please add a filter entry to block this popup?

**Comment by BillyBobMcInbredYahoo** [comment]
Boss-like attitude? What boss-like attitude? I wrote "Could someone please add a filter entry to block this popup?". There's nothing boss-like about that. I also thank people when they help me.

YOU on the other hand... You've marked issues I've raised as completed when they weren't and marked them as duplicates when they weren't. And if it was you that marked one of my comments as spam, that was totally uncalled for.

Other people have been helpful, but not you. You're more of a hindrance than a help. Why you have the privileges you do on this site is mind-boggling.

**Comment by BillyBobMcInbredYahoo** [comment]
Plus at the top of the screen, the issue 25063 is marked as 'Closed', so why would anyone read the entire thread and see the NEW issue 25166 that I raised near the bottom of the thread? Answer: They wouldn't.

This is precisely why I created a new issue #25166, but I was told that was the wrong way to go about things and to add my new issue to this existing thread that is now mark as 'Closed'.

This is utterly nonsensical, which I explained in my comment that was marked as 'Spam'. Which has now been labelled instead as 'off-topic' even though it's completely ON topic.

**Comment by garry-ut99** [comment]
If there are several similiar issues on the same site, especially in a short period of time, the principle is to keep them reported in a single thread in order not to spam the issue tracker, and for the sake of easier management for developers. (even if they are not 100% identical, they are similiar enough to be qualified into category of duplicates).

**Comment by BillyBobMcInbredYahoo** [comment]
I get that (although it's unintuitive), but this issue has been marked as 'Closed' when there's an unresolved pop-up. How do I get this issue re-opened?

**Comment by BillyBobMcInbredYahoo** [comment]
I've been waiting TWO DAYS.

Stop messing with me. Go away.

Never reply to me again.
Never edit anything relating to this issue again.

**Comment by stephenhawk8054** [comment]
We are volunteers, not your employees.

If volunteers are busy to address your issue, report to this repo about the nuisances: https://github.com/easylist/easylist/issues
